src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,119 @@
+package duke.task;
+
+import duke.exception.DukeException;
+
+import java.util.ArrayList;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+/**
+ * Contains the task list and basic operations involving it.
+ */
+public class TaskList {
+    // tasklist exception messages
+    private final static String INDEX_NO_EXIST = "Yo yo yo, this task don't exist. Give a legitimate task number.";
```

Comment:
Could be changed to INVALID_INDEX_TEXT instead?

Reply:
Thanks for the suggestion! I'll change it!